src/app/core/header/header.component.spec.ts

```diff
@@ -22,4 +26,12 @@ describe('HeaderComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should be initialized properly', () => {
+    const component = new HeaderComponent();
```

Comment:
this is wrong, you don't create components like that, it will initialize instance of the class, but it won't be real component + if it has dependencies you will have to provide them here, better way on line [22](https://github.com/vyarmolovich/video-portal/pull/3/files#diff-a87ce1267f15d99da7b6c8c41472da83R22)

so you can just skip you re-'initialization' and check properties on already created component